examples/replay/replay_mover.py

```diff
@@ -0,0 +1,405 @@
+                        for _ in range(len(xds["time"])//2)
+                    ]
+                ).flatten()
+
+        xds["ftime"] = xr.DataArray(
+                ftime,
+                coords=xds["time"].coords,
+                dims=xds["time"].dims,
+                attrs={
+                    "axis": "T",
+                    "description": "time passed since forecast initialization",
+                    "long_name": "forecast_time",
+                    },
+                )
+        xds["cftime"] = xr.DataArray(
+                time2cftime(xds["time"]),
+                coords=xds["time"].coords,
+                dims=xds["time"].dims,
+                attrs={
+                    "calendar_type": "JULIAN",
+                    "cartesian_axis": "T",
+                    "long_name": "time",
+                    },
+                )
+        xds = xds.set_coords(["ftime", "cftime"])
+        return xds
+
+
+class ReplayMoverQuarterDegree(ReplayMover1Degree):
+
+    @property
+    def xcycles(self):
+        """These are the DA cycle timestamps, which are every 6 hours. There is one s3 directory per cycle for replay."""
+        cycles = pd.date_range(start="1994-01-01", end="2023-10-13T06:00:00", freq="6h")
+        return xr.DataArray(cycles, coords={"cycles": cycles}, dims="cycles")
+
+
+    @property
+    def xtime(self):
+        """These are the time stamps of the resulting dataset, assuming we are grabbing fhr00 and fhr03"""
+        time = pd.date_range(start="1994-01-01", end="2023-10-13T09:00:00", freq="3h")
+        iau_time = time - timedelta(hours=6)
+        return xr.DataArray(iau_time, coords={"time": iau_time}, dims="time", attrs={"long_name": "time", "axis": "T"})
+
+    def __init__(
+        self,
+        n_jobs,
+        config_filename,
+        component="fv3",
+        storage_options=None,
+        main_cache_path=f"/contrib/Tim.Smith/tmp-replay/0.25-degree",
+    ):
+        super().__init__(
+            n_jobs=n_jobs,
+            config_filename=config_filename,
+            component=component,
+            storage_options=storage_options,
+            main_cache_path=main_cache_path,
+        )
+
+    @staticmethod
+    def cached_path(dates, forecast_hours, file_prefixes):
+        upper = "filecache::s3://noaa-ufs-gefsv13replay-pds"
+        dates = [dates] if not isinstance(dates, Iterable) else dates
+
+        files = []
+        for date in dates:
+            this_dir = f"{date.year:04d}/{date.month:02d}/{date.year:04d}{date.month:02d}{date.day:02d}{date.hour:02d}"
+            for fp in file_prefixes:
+                for fhr in forecast_hours:
+                    this_file = join(this_dir, f"{fp}{date.year:04d}{date.month:02d}{date.day:02d}{date.hour:02d}_fhr{fhr:02d}_control")
+                    files.append(this_file)
+        return [join(upper, this_file) for this_file in files]
+
+
+def batched(iterable, n):
```

Comment:
I recall seeing this function in the repository before. Would it be possible to retain one version of the function?

Reply:
Good call, this can be imported with 

```python
from ufs2arco.utils import batched
```

but since it's no longer used in the replay mover code, since running in batches of multiple cycles at a time was no faster and less resilient, I'll just delete it from this module.

Reply:
Done with 
34178a6